src/AaveV3_Eth_AaveV3ListRPL_20230711_20231107/AaveV3_Eth_AaveV3ListRPL_20230711_20231107.t.sol

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import 'forge-std/Test.sol';
+import {GovHelpers} from 'aave-helpers/GovHelpers.sol';
+import {AaveGovernanceV2} from 'aave-address-book/AaveGovernanceV2.sol';
+import {AaveV3Ethereum, AaveV3EthereumAssets} from 'aave-address-book/AaveV3Ethereum.sol';
+import {ProtocolV3TestBase, ReserveConfig} from 'aave-helpers/ProtocolV3TestBase.sol';
+import {AaveV3_Eth_AaveV3ListRPL_20230711_20231107} from './AaveV3_Eth_AaveV3ListRPL_20230711_20231107.sol';
+
+/**
+ * @dev Test for AaveV3_Eth_AaveV3ListRPL_20230711_20231107
+ * command: make test-contract filter=AaveV3_Eth_AaveV3ListRPL_20230711_20231107
+ */
+contract AaveV3_Eth_AaveV3ListRPL_20230711_20231107_Test is ProtocolV3TestBase {
+  address public constant RPL_USD_FEED = 0x4E155eD98aFE9034b7A5962f6C84c86d869daA9d;
+  address public constant RPL = 0xD33526068D116cE69F19A9ee46F0bd304F21A51f;
+
+  function setUp() public {
+    vm.createSelectFork(vm.rpcUrl('mainnet'), 17669147);
+  }
+
+  function testProposalExecution() public {
+    AaveV3_Eth_AaveV3ListRPL_20230711_20231107 proposal = new AaveV3_Eth_AaveV3ListRPL_20230711_20231107();
+
+    ReserveConfig[] memory allConfigsBefore = createConfigurationSnapshot(
+      'preAaveV3_Eth_AaveV3ListRPL_20230711_20231107',
+      AaveV3Ethereum.POOL
+    );
+
+    GovHelpers.executePayload(vm, address(proposal), AaveGovernanceV2.SHORT_EXECUTOR);
+
+    ReserveConfig[] memory allConfigsAfter = createConfigurationSnapshot(
+      'PostAaveV3_Eth_AaveV3ListRPL_20230711_20231107',
+      AaveV3Ethereum.POOL
+    );
+
+    // RPL
+
+    ReserveConfig memory RPL = ReserveConfig({
```

Comment:
Maybe we can avoid shadowing the existing variable RPL which is declared as constant above.

Reply:
? not sure of what is suggested here.

Reply:
hi Marc, there are two variables with the name `RPL` on the code, which is known as variable shadowing. This can be confusing as it's hard to tell which variable is being referenced. Depending on the language, there might be different behaviors.

https://en.wikipedia.org/wiki/Variable_shadowing

Reply:
Resolved. :)

Reply:
thx fermin for feedback, solved it!